start

```diff
@@ -22,484 +22,544 @@ export PGPORT=5432
 
 QUICKSTART_INITIALIZED=false
 CURRENT_POSTGRES_PID=""
+QUICKSTART_LOG="/quickstart.log"
+
 
 function main() {
-  echo ""
-  echo "Starting Stellar Quickstart"
-  echo ""
-
-  process_args $*
-  if [ "$NETWORK" != "standalone" ] && [ "$ENABLE_HORIZON_CAPTIVE_CORE" = "true" ]; then
-    echo "--enable-horizon-captive-core usage only supported on standalone networks" >&2
-    exit 1
-  fi
-  if [ "$NETWORK" != "standalone" ] && [ "$NETWORK" != "futurenet" ] && [ "$ENABLE_SOROBAN_RPC" = "true" ]; then
-    echo "--enable-soroban-rpc usage only supported on standalone/futurenet networks" >&2
-    exit 1
-  fi
-  if [ "$NETWORK" != "standalone" ] && [ "$ENABLE_CORE_ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING" = "true" ]; then
-    echo "--enable-core-artificially-accelerate-time-for-testing is only supported in the standalone network" >&2
-    exit 1
-  fi
-  if [ "$NETWORK" != "standalone" ] && [ "$RANDOMIZE_NETWORK_PASSPHRASE" = "true" ]; then
-    echo "--randomize-network-passphrase is only supported in the standalone network" >&2
-    exit 1
-  fi
-  if [ "$ENABLE_HORIZON_CAPTIVE_CORE" = "true" ] && [ "$ENABLE_CORE_ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING" = "true" ]; then
-    echo "--enable-core-artificially-accelerate-time-for-testing is only supported without captive core" >&2
-    exit 1
-  fi
-
-  echo "mode: $STELLAR_MODE"
-  echo "network: $NETWORK ($NETWORK_PASSPHRASE)"
-
-  copy_defaults
-  init_db
-  init_stellar_core
-  init_horizon
-  copy_pgpass
-  init_soroban_rpc
-
-  stop_postgres  # this gets started in init_db
-
-  # launch services
-  exec_supervisor
+	print_screen_output "" 
```

Comment:
Almost every line is this file is changed because the style is changing. Could we keep the existing style? It seems pretty out of scope of this PR to make a broad style change.

Reply:
I reverted formatting back.